zstacklib/zstacklib/utils/objgraph.py

```
@@ -0,0 +1,960 @@
+    InstanceType = None
+
+
+__author__ = "Marius Gedminas (marius@gedmin.as)"
+__copyright__ = "Copyright (c) 2008-2016 Marius Gedminas and contributors"
+__license__ = "MIT"
+__version__ = "3.0.0"
+__date__ = "2016-04-13"
+
+
+try:
+    basestring
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)